EventFilter/SiPixelRawToDigi/src/PixelDataFormatter.cc

```diff
@@ -191,26 +211,61 @@ void PixelDataFormatter::interpretRawData(bool& errorsInEvent, int fedId, const
       detDigis = &digis.find_or_insert(rawId);
       if ( (*detDigis).empty() ) (*detDigis).data.reserve(32); // avoid the first relocations
     }
-    if unlikely(skipROC) continue;
-    
+
+    // skip is roc to be skipped ot invalid
+    if unlikely(skipROC || !rocp) continue;
     
-    int dcol = (ww >> DCOL_shift) & DCOL_mask;
-    int pxid = (ww >> PXID_shift) & PXID_mask;
     int adc  = (ww >> ADC_shift) & ADC_mask;
-    
-    LocalPixel::DcolPxid local = { dcol, pxid };
-    if unlikely(!local.valid()) {
-      LogDebug("PixelDataFormatter::interpretRawData") 
-	<< "status #3";
-      errorsInEvent = true;
-      errorcheck.conversionError(fedId, &converter, 3, ww, errors);
-      continue;
-    }
-    
-    GlobalPixel global = rocp->toGlobal( LocalPixel(local) );
+    //LocalPixel *local=NULL;
```

Comment:
It would be nice to delete obsolete commented lines like this. Previous versions are stored in Git, so there's no need to clutter the file with them.